test/functional/tests/fault_injection/test_fault_injection_interrupts.py

```diff
@@ -236,11 +236,11 @@ def test_interrupt_core_remove(cache_mode, filesystem):
             flush_pid = TestRun.executor.run_in_background(
                 cli.remove_core_cmd(str(cache.cache_id), str(core.core_id)))
 
-        with TestRun.step("Interrupt core removing"):
+        with TestRun.step("Flush core."):
```

Comment:
I think the original name was pretty accurate. Why do you want to change it?

Reply:
You are right, my fault I have unintentionally changed it.